cmd/utils/flags.go

```diff
@@ -533,8 +533,8 @@ var (
 
 	EnvironmentFlag = Flag{
 		Name:  c_NodeFlagPrefix + "environment",
-		Value: params.LocalName,
-		Usage: "environment to run in (local, colosseum, garden, orchard, lighthouse, dev)" + generateEnvDoc(c_NodeFlagPrefix+"environment"),
+		Value: params.ColosseumName,
```

Comment:
I would like to add a print message somewhere indicating which network is being run, that would presumably also have solved your problem of not realizing the default was local.

Reply:
fixed